Integrations/integration-CiscoSpark.yml

```diff
@@ -0,0 +1,751 @@
AACjVqt2TeeYWZIVW87P/mbtpWiR8/oEkXaUEIB6+KX+NhWmiczlkS80d2vcrjLOYQIAqiICAD3922TlzPmLd3/0w280tI239Ms3PnOy4G3XBg0bcZcHAAC4h5gMZd7rHz4nlp9XitwTPiwVSP3jm//MY+PE01cSxJIXUk/b2NrYVta5SwBAVVQA8KfnLr97jeUBjfftv37lhslhigju7AAAAPdhZ2drX8PJsZZL/dqNNx9Z9O8X3pwr9PMDZOkaecUTBRYPD6DhS40s7iEy6/WJ4pmCNJG3LnWXvYOdY2WeuwQAVEUHAI41nJymaxNTCQGMr/QT/00MU6htbG1tubMDAAA8hPadW/cYNKqXd+ZTmh07ry6++UxBiph3aqpIO0IIQI0Uc076ic2F6SJjsXZTk2YNW1b2+UoAQFV0AKDXplOXboc//ObGHkIAo+r6f+yLm6Jjj959uJMDAAA8hllLUzY8f2rpx4teiby+8mKkWHJOKuafmi5mvT5JpB8dTYO4inX7zznuLdZdjhPLdqads7axtjGGc5QAgKqMAEBvwAiP0S++93URPQGMo+v//k+KROzs/MVW1tbW3L0BAACegO55ysZ91MCxm/YvP3Xkg2fFljdmioVnQ8XM1yeJ7OPj7ywlyOSBZt3tX3+c119JEPsvbvzE0dHBaJbWIgCgKisA0IvMzJ332uc/iZ3XbtIQr8Tade2G2Hrh7Q+5WwMAAJQhB0f76i6uzg1q163ZwGPikGl5G5JfevndtWJDQaqYfXJK8VviO5MHEgaYU+knhtxUkCZyV8a/4FSzem1jOifDJZJkGrxUZQUA+rHmgzzGeh374idBT4DKq+cL3v+kUfOWrbhLAwAAlKMmzRu1XPbsnJdy1mpf0i71eWnVybjvnr4SK5ael4rZJ6YUTyBIA9q0u/3rh3usuxInlmxPPTNq3NDJxnYO7t675180eKnKCgD0rHSic/MXb/+/L37cQQhQ4V3/916/KUITMnK4IwMAAFQwl7rO9dbtWHh4z8VVX648Fv/lktPyXxecCRJzTvqKnOMTRDqrCZhQt3+P4tn+9d3+V+/NPW9ra2NnjOccAQBV2QHAn1bse+3krve//ak4BPjoBg30Cun6XyRe/ui7m6OnTJ3OHRgAAKCS9R7QfciC9Rl7Xnl3ndhYkCzmnPh7acH0o55/DBegsW2cb/4ni80F6WLF8zOPWuoY6zlGAEAZSwCgl7x0zcYDn955M00Dvfxr3ydFYsgYrwncbQEAAIxE46YNm2tSwueGxU+e668dPnfT2ZnfbHkjQ6y6GFE8TEA/gWAq8wYYVc0+MVlsLEgSuWtj9nlOGOZnzOcXAQBlTAFAq/YdOuU9u3Pf3us3CAHKufZcLxLPnX/7fZd69RtwpwUAADBSXXt27Ltux8IjC7YmH0leG3Rkw8W0n9ZcjhGLz4WK3BOTRCbzBlRa6XtlzDsVILZczRbr98+/YMxv/gkAKGMMAPQsdL+b1YdOXNh7/Y+eAB/RWC/r0g+zeOnj739p1LR5C+6qAAAAJqRx0wYt1Ckhcw++u6Z4jfm8kwHF8wVkHRsrMvTDBJg3oMLG/OtXcnj+zTli66srC2ztbO1N4fwhAKCMLQDQs7Wzs1tz9MKbu6/p5wJgPoCynvhvz/UbIn7Rqg3cQQEAAExQXdc69cf4DJ8+3Lvf9PAE37kr92cX7Hg7T6y8qBF5pwJExrExxRMIph3xYKhAmdcokXF0jJh/OlDsLlz2g2/YOG2zlk1am8q5QwBAGWMAoDdghOeYhdtfOrznGgFAmb7919WmUwVvt+vWoxd3TwAAADPQuFnDFnEzlUvkaf5LsterD71QOP+nTQWpt1acVxTPG8BqAmXV5d9DzD3pJ9ZfSbr14psriwYO7zPa1M4VAgDKWAMAvVbtO3aa/8L+V3fScC+T2nmtSOz98Ptf+rgPH8mdEgAAwIw9e2D5uVff2yA2FCSL+WcCi4OAma97i8xjY0TaUXoFPOos//rJF+ee9BebrqaKlbtyT5nqeUEAQBlzAKDXtJVb6xc//uG3HTTgn7BuCP28CtvOvfkOd0QAAAAz17xl09atO7To2qJd464de7u57z6/5vq+/1suVl+KFLNP+vyxtKBH8Vh2AoEHv/XPPjZBrLkUI45/+PztQaN6TWjaopEbAQBFAFB+ug0Y7P584YefsTLAE7z9//iG2HL5nevNW7dtxx0RAACgimnaonGrSQGjJR5+AySRs6avWXc4590X3pojnroYKeaemlr8lpsGv2HpV1l4+kqcWHs4+73JYR4xQ0cNHGfq5wEBAGUKAYDeQM9x47ddee86IcDj1cufFomQ2JQM7n4AAACo1qFL2+4XP3715qH3nhFb35gplp6X/TVUIPuYl0g9WlW7+3uImScmivzTM4qHT7z+0XPCY8JQX3M57gQAlKkEAHoTZoRJX/zkJ8FwgEed+O+GePm9L76vWbu2M3c7AAAAFLP4SzWLBo3rNV23N//0S++sEU9fSRR5p6aK3BOTRc7r40XW8bEi/egoXePYXJf0uzPGXz9HQv7pQPHCW3NFzsqYHfr9Urx3zAgBAGVKAUDDps2abzp2vmDftR9/169lT+P+4Wb9P/DpDTFdHRXHXQ4AAAD3ZWVlZd25R4e+Hbq37hui8U059Nbm71789xLx9OU4kXPcW6Qf8TDDxr9H8batuxIr1h3N+ihmpmRZxx5t+9Zv6NrYHI8xAQBlSgHAn1KXrd348mc3aeA/ROkn/guMjE20sbW1464GAACAh9apa/seC9dn7dr4St6beXsi31x/Ienm6ksaseBMkMg67lXceDbNif08xZwTU8TS81Kx+nzcL3N2qd5c/8rcNxNmqhab+zElAKBMMQBwqlXbeeWBY2de/OSmYE6ABy/7t+fdL39o0qJlK+5gAAAAeCJePh4Bu06s/b+X9cMELieIpeekxV3m9asKZB/3Ehm6hrXRzuSv+36zT/iIvFMBxd/7+X/l
```

Comment:
you can do this by using replaceInTemplatesAndRemove,
The url include url/%key% and args key=value, so after using it you will have url/value